pyvelox/pyvelox.h

```diff
@@ -504,6 +527,56 @@ static void addVectorBindings(
             std::move(baseVector),
             PyVeloxContext::getSingletonInstance().pool());
       });
+
+  m.def(
+      "row_vector",
+      [](std::vector<std::string>& names,
+         std::vector<VectorPtr>& values,
```

Comment:
nit: values seems like bad name, lets name it children ?
Should this be a const reference or is the idea that these get moved into the rowvector ?

Reply:
I think it would be better to move? WDYT?